srcpkgs/perl-OpenGL/template

```diff
@@ -0,0 +1,35 @@
+# Template file for 'perl-OpenGL'
+pkgname=perl-OpenGL
+version=0.70
+revision=1
+#wrksrc=
+#create_wrksrc=yes
+#only_for_archs="i686 x86_64"
+build_style=perl-module
+#configure_args=""
+#make_build_args=""
+#make_install_args=""
+#conf_files=""
+#make_dirs="/var/log/dir 0755 root root"
+hostmakedepends=""
+makedepends="perl libfreeglut-devel libXmu-devel"
+depends="perl"
+short_desc="Wrapper library that allows OpenGL to be used in Perl"
+maintainer="Jasper Chan <jasperchan515@gmail.com>"
+license="GPL-3.0-or-later"
+homepage="https://metacpan.org/pod/release/CHM/OpenGL-0.70/OpenGL.pod"
+#distfiles="https://cpan.metacpan.org/authors/id/C/CH/CHM/OpenGL-0.70.tar.gz>perl-OpenGL-0.70.tar.gz"
+distfiles="https://github.com/Gigahawk/perl-OpenGL/archive/0.70_4.tar.gz>perl-OpenGL-0.70.tar.gz"
```

Comment:
Why are you rehosting some of the packages instead of using patches and fetching the official sources?